### PR TITLE
zh-cn: fix the text style and complete the translation

### PR DESCRIPTION
**Pull request by UGdango**
改正样式，翻译少量句子

### Description
Fix text style error, translate a sequence

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/proxy/proxy/has/index.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ var p = new Proxy(target, {
 
 ### 参数
 
-`下面是传递给 has` 方法的参数。`this` is bound to the handler.
+下面是传递给 has 方法的参数。`this` 绑定至 handler 对象上。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/proxy/proxy/has/index.md:22]:
```suggestion
下面是传递给 `has()` 方法的参数。`this` 会被绑定至 handler 上。
```

**Review by yin1999 — COMMENTED** [review]
Thank you @UGdango. Could we follow the style of [the en-US documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy/Proxy/has)

File: files/zh-cn/web/javascript/reference/global_objects/proxy/proxy/has/index.md (hunk 1 above)
Comment (UGdango) [reply files/zh-cn/web/javascript/reference/global_objects/proxy/proxy/has/index.md:22]: sure!

**Comment by UGdango** [comment]
> Thank you @UGdango. Could we follow the style of [the en-US documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy/Proxy/has)

sure!

File: files/zh-cn/web/javascript/reference/global_objects/proxy/proxy/has/index.md (hunk 1 above)
Comment (UGdango) [reply files/zh-cn/web/javascript/reference/global_objects/proxy/proxy/has/index.md:22]: ok

**Review by yin1999 — APPROVED** [review]
Thanks @UGdango. This PR has been open for two weeks, so I allowed myself to commit the suggestion, and then we can merge this :)

Welcome to the community :tada:

**Pull request merged**